data/lead.json

```diff
@@ -222,7 +223,7 @@
 	{
 		"name": "Stoking the Fire",
 		"grade": "9b",
-		"fa": "Adam Ondra",
+		"fa": "Chris Sharma",
 		"repeat": ["Stefano Ghisolfi", "Sachi Amma"],
```

Comment:
Thanks for spotting, could you please add Adam Ondra to the repeat row? He actually did the [2nd ascent](https://www.ukclimbing.com/news/2016/02/stoking_the_fire_9b_by_ondra-70276).

Reply:
Done! How did I missed that? :)